_translator-files/po/de/Installation-for-Android.po

```diff
@@ -51,7 +53,7 @@ msgstr "# Installation auf Android"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Android.md:14
 msgid "Make sure you've already read the [Getting Started](Getting-Started) page."
-msgstr "Wenn du die [Erste Schritte](Getting-Started) Seite noch nicht gelesen hast, solltest du das nachholen."
+msgstr "Vergewissere dich, dass du die Seite [Erste Schritte](Getting-Started) bereits gelesen hast."
```

Comment:
I'd still prefer the old translation - it sounds a bit more natural, IMHO

Reply:
Changed to former text
`Wenn du die [Erste Schritte](Getting-Started) Seite noch nicht gelesen hast, solltest du das nachholen.`